groups/models.py

```diff
@@ -1,20 +1,57 @@
 from django.conf import settings
+from django.contrib.auth import get_user_model
 from django.core.urlresolvers import reverse
 from django.db import models
 from django.template import loader, RequestContext
 from django.utils import timezone
-from polymorphic import PolymorphicManager, PolymorphicModel
+from polymorphic import PolymorphicManager, PolymorphicModel, PolymorphicQuerySet
 
 
-class DiscussionManager(models.Manager):
+class GroupQuerySet(models.QuerySet):
+    """A queryset for Groups allowing for smarter retrieval of related objects."""
+    def discussions(self):
```

Comment:
It pretty visibly does that (and that's what you'd expect a method called `discussions` on a `GroupQuerySet` to do).  Might be worth clarifying what the relation actually means though, I suppose.